src/Protocol/Protocol18x.cpp

```diff
@@ -3415,6 +3411,9 @@ void cProtocol180::WriteMobMetadata(cPacketizer & a_Pkt, const cMonster & a_Mob)
 			a_Pkt.WriteBEInt32(Appearance);
 			a_Pkt.WriteBEUInt8(0x56);  // Int at index 22
 			a_Pkt.WriteBEInt32(Horse.GetHorseArmour());
+            
+            a_Pkt.WriteBEUInt8(0x0c);
```

Comment:
We use tabs at the start of a line.

Please use the CheckBasicStyle.lua file in the src folder to find most of the style errors.